docs/anomaly-detection/guides/guide-vmanomaly-vmalert.md

```diff
@@ -196,6 +202,9 @@ In the query expression we need to put a condition on the generated anomaly scor
 You can choose your threshold value that you consider reasonable based on the anomaly score metric, generated by vmanomaly. One of the best ways is to estimate it visually, by plotting the `anomaly_score` metric, along with predicted "expected" range of `yhat_lower` and `yhat_upper`. Later in this tutorial we will show an example
 
 ## 8. Docker Compose configuration
+
+You can find the `docker-compose.yml` and all configs in this [folder](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/vmanomaly/vmanomaly-vmalert-guide/)
+
 Now we are going to configure the `docker-compose.yml` file to run all needed services.
```

Comment:
I assume with the addition of alertmanager, current scheme of docker-compose should have 7 elements (instead of 6)
![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/11490990/9d22bdab-6a20-41ac-8b5e-fe7d185f7956)

Reply:
i will just remove it, this pictire doesn't make any sense to me